systers_portal/templates/account/email_confirm.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% load account %}
+
+{% block title %} - Confirm E-mail Address{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <h2>Confirm E-mail Address</h2>
+    <hr>
+    {% if confirmation %}
+      {% user_display confirmation.email_address.user as user_display %}        
+      <p>Please confirm that <a href="mailto:{{ confirmation.email_address.email }}">{{ confirmation.email_address.email }}</a> is an e-mail address for user {{ user_display }}.</p>
+      <form method="post" action="{% url 'account_confirm_email' confirmation.key %}">
+      {% csrf_token %}
+        <button type="submit">Confirm</button>
```

Comment:
Please style the button as primary. You can see how it is done by looking at other submit-confirm-type buttons throughout the project.